frontend/src/components/auth/Join.tsx

```diff
@@ -45,13 +46,18 @@ const PLACEHOLDER_WEB_EXAMPLE_FULL_NAME = "Jane Doe";
 const PLACEHOLDER_WEB_EXAMPLE_PHONE_NUMBER = "111-222-3333";
 const PLACEHOLDER_WEB_EXAMPLE_EMAIL = "example@domain.com";
 const PLACEHOLDER_WEB_EXAMPLE_ORG_NAME = "Feeding Canadian Kids";
+const PLACEHOLDER_WEB_EXAMPLE_NUM_KIDS = "50";
 const PLACEHOLDER_WEB_EXAMPLE_ADDRESS = "123 Main Street, Anytown";
+const PLACEHOLDER_WEB_EXAMPLE_DESCRIPTION =
+  "Non-Profit Organization in Alberta";
```

Comment:
number of kids and org description placeholders differ from the placeholders in User Settings. we should probably standardize so that they are the same.